### ISSUE TITLE
Any plans to modernize this?

### ISSUE THREAD

**Issue by furudean** [kobaltcore/renconstruct-action#1]
I've built my own action for renkit but it would nice if this was supported out of the box too!

**Comment by furudean** [comment]
Nevermind, I just don't know how to read instructions

**Comment by kobaltcore** [comment]
Yeah, this could probably stand to be looked at again, I'll see if I can take a look at this soon, it's languished for a while by now.

**Comment by furudean** [comment]
Wait... does this or does this _not_ support modern renkit? I remember trying this action and running into some sort of issue but now I'm unsure

**Comment by kobaltcore** [comment]
It might support the new versions by accident, but certainly not on purpose. I'll definitely have to take another look to make sure everything is kosher.
